include/repgp/repgp_def.h

```diff
@@ -266,7 +296,7 @@ typedef enum {
                                * IETF-S/MIME) */
     PGP_PKA_EDDSA = 22,       /* EdDSA from draft-ietf-openpgp-rfc4880bis */
 
-    PGP_PKA_SM2 = 99,         /* SM2 encryption/signature schemes */
+    PGP_PKA_SM2 = 99, /* SM2 encryption/signature schemes */
```

Comment:
Missing tab for aligning comments?

Reply:
@ronaldtse I 99% sure that it is a side effect of clang-based reformating.